ext/dom/dom.stub.php

```diff
@@ -1,5 +1,28 @@
 <?php
 
+interface DOMChildNode
+{
+    public function remove() : void;
+
+    /** @var ...DOMNode|string|null $nodes */
```

Comment:
The `|null` here and below looks wrong. It doesn't seem to be accepted by your implementation, and also not by the spec.